x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/alerts_list.ts

```diff
@@ -640,24 +639,34 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       // Select enabled
       await testSubjects.click('ruleStatusFilterButton');
       await testSubjects.click('ruleStatusFilterOption-enabled');
+      await testSubjects.waitForDeleted('.euiBasicTable-loading');
+
       await assertRulesLength(1);
 
       // Select disabled
       await testSubjects.click('ruleStatusFilterOption-enabled');
       await testSubjects.click('ruleStatusFilterOption-disabled');
+      await testSubjects.waitForDeleted('.euiBasicTable-loading');
+
       await assertRulesLength(1);
 
       // Select snoozed
       await testSubjects.click('ruleStatusFilterOption-disabled');
       await testSubjects.click('ruleStatusFilterOption-snoozed');
+      await testSubjects.waitForDeleted('.euiBasicTable-loading');
+
       await assertRulesLength(1);
 
       // Select disabled and snoozed
       await testSubjects.click('ruleStatusFilterOption-disabled');
+      await testSubjects.waitForDeleted('.euiBasicTable-loading');
```

Comment:
This is a css-selector, not a test-subject selector, so I think you want

```ts
find.waitForDeletedByCssSelector('.euiBasicTable-loading');
```

Reply:
my bad, saw that waitForDeleted calls waitForDeletedByCssSelector, thought it would implicitly differentiate the selectors

Reply:
Ah, that actually translates the testSubjectSelector into a css selector before using that method